CRM/CiviAwards/Service/AwardProfile.php

```diff
@@ -53,7 +63,25 @@ public function createProfileFields(array $customFields, $profileId) {
     $this->addContactIdField($profileId);
   }
 
-    $this->addContactIdField($profileId);
+  /**
+   * Validates review_field options to ensure legal options are passed.
+   *
+   * @param array
+   *  Custom field
+   *
+   * @throws \CiviCRM_API3_Exception
+   */
+  private function validateReviewFields($customField) {
```

Comment:
No function type hinting for $customField? 
Please make sure to run the npm install and the pre-commit hook. Things like this are not supposed to be brought up in code review because the linters should have taken care of that already.

Thanks